webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2427,6 +2433,8 @@ var CURRENT_GROUP_DATA={};
                         $applozic(this).hide();
                     }
                     clearTimeout(MCK_TRIGGER_MSG_NOTIFICATION_PARAM);
+                    clearTimeout(MCK_CHAT_POPUP_TEMPLATE_TIMER);
```

Comment:
@CaptainT33mo these 3 lines are common at multiple places, this can be moved to a single function

Reply:
@devashishmamgain done.